e2e/exhaustive_suite_test.go

```diff
@@ -60,17 +33,30 @@ func Exhaustive(dt *testing.T) {
 		}
 	})
 
-	generateWorkloads(Workloads)
+	pvcSpecs := config.GetPVCSpecs()
+
+	for _, tc := range config.GetTests() {
+		pvcSpec, ok := pvcSpecs[tc.PVCSpec]
+		if !ok {
+			t.Fatalf("unknown pvcspec: %q", tc.PVCSpec)
+		}
```

Comment:
I think we need to separate validation of the config from running the tests, so wrong config value will fail early before starting any tests.

I this should work:

1. Iterate on the config, validating the values and appending tests to list
2. Iterate on validated tests and run them

Step 1 should be done in the config module.